worker/test/src/RTC/TestSeqManager.cpp

```diff
@@ -1400,4 +1400,19 @@ SCENARIO("SeqManager", "[rtc][SeqMananger]")
 		SeqManager<uint16_t> seqManager(/*initialOutput*/ 1000);
 		validate(seqManager, inputs);
 	}
+
+	SECTION("map packets prior to first mapped packet")
```

Comment:
I fail to see the value  that this test adds to the already existing tests? Isn't this already tested in existing tests?

Also, what does "mapped packet" mean? We don't use that terminology anywhere in the source code and in existing tests.

Reply:
Not covered (or at least not explicitily), does backward wrap-around as a bonus. What terminology do you suggest?